### PR TITLE
Add Article and ArticleList

### PR DESCRIPTION
**Pull request by Gallaecio**
To do:
- [x] Clarify the schema of `Article.authors[*]`.
- [x] Complete the implementation.

### CODE REVIEW

**Review by kmike — APPROVED** [review]
👍

File: zyte_common_items/items.py [hunk 1]
```diff
@@ -20,10 +23,226 @@
...
+    #: In case there is no article data on the page or the page was not
+    #: reached, the returned "empty" item would still contain this URL field.
+    url: str = attrs.field(converter=url_to_str)
+
+    #: Data extraction process metadata.
+    metadata: Optional[DateDownloadedMetadata] = None
+
+
+@attrs.define(slots=True, kw_only=True)
+class ArticleList(Item):
```
Comment (kmike) [inline zyte_common_items/items.py:199]: If I'm not mistaken, we should keep only url, articles and metadata fields.

**Review by kmike — CHANGES_REQUESTED** [review]
We should remove extra fields from the schemas.

File: zyte_common_items/items.py (hunk 1 above)
Comment (Gallaecio) [reply zyte_common_items/items.py:199]: I’ve removed paginationNext and pageNumber], I think the rest may be meant to stay, even breadcrumbs.

**Pull request merged**